components/lists/SessionsListVertical.tsx

```diff
@@ -0,0 +1,95 @@
+import { useTheme } from '@react-navigation/native';
+import { useRouter } from 'expo-router';
+import React from 'react';
+import type { ListRenderItemInfo } from 'react-native';
+import { FlatList, StyleSheet, View } from 'react-native';
+import SessionListSeparator from '../../assets/artworks/ListSeparator';
+import type { SessionForSchedule } from '../../global';
+import SessionCard from '../cards/SessionCard';
+import Row from '../common/Row';
+import Space from '../common/Space';
+import StyledText from '../common/StyledText';
+
+interface SessionListVerticalProps<T> {
+  variant?: 'card' | 'list';
+  bookmarked: boolean;
+  handleBookMark?: (id: number) => void;
+  sessions: Array<T>;
+}
+
+/**
+ * @param variant: 'card' | 'list'
+ * @param bookmarked: boolean
+ * @param handleBookMark: () => void
+ * @param sessions: Array of type Sessions
+ */
+const SessionsListVertical = ({
+  variant = 'card',
+  bookmarked = false,
+  sessions,
+  handleBookMark,
+}: SessionListVerticalProps<SessionForSchedule>) => {
+  const { colors } = useTheme();
+  const router = useRouter();
+
+  const renderComponentItem = ({ item, index }: ListRenderItemInfo<SessionForSchedule>) => {
+    return (
+      <View key={item.id}>
+        <SessionCard
+          handlePress={() => router.replace({ pathname: `/session/${item.slug}`, params: { slug: item.slug } })}
```

Comment:
Most of the sessions in the `schedule.ts` mock file have an empty slug. This crashes the app on Android when navigating to the individual session screen and navigates to a broken session screen on iOS and the back button does not work. We should add a check for this and fix up the mock data as well to provide the slugs.

Reply:
Uhm, what if we disable session card from being pressed when a session has an empty slug? quick fix innit?